lib/ClangImporter/ClangImporter.cpp

```diff
@@ -68,7 +68,9 @@ using namespace importer;
 using clang::CompilerInstance;
 using clang::CompilerInvocation;
 
+#if !defined(_MSC_VER)
 #pragma mark Internal data structures
+#endif
```

Comment:
I can... I'm personally not a fan of disabling warnings, they exist for a reason. That said, considering MSVC is "pretty awful", it could make sense